src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java

```diff
@@ -176,6 +177,15 @@ else if (getServer().getPluginManager().isPluginEnabled("CS-CoreLib")) {
                 return;
             }
 
+            // Disabling backwards-compatibility for fresh Slimefun installs
+            if (!new File("plugins/" + getName().replace(" ", "_"), "config.yml").exists()) {
+                config = new Config(this);
+                config.setValue("options.backwards-compatibility", false);
+                config.save();
```

Comment:
I'm thinking maybe data storage should be checked. Since that's a good indication of a new server and it's w good way to see if there's existing items.

Reply:
Hmm, good idea